bcos-txpool/bcos-txpool/txpool/storage/MemoryStorage.cpp

```diff
@@ -79,6 +79,34 @@ void MemoryStorage::stop()
 
 task::Task<protocol::TransactionSubmitResult::Ptr> MemoryStorage::submitTransaction(
     protocol::Transaction::Ptr transaction)
+{
+    co_return co_await submitTransactionWithHook(transaction, nullptr);
+}
+
+std::vector<protocol::Transaction::ConstPtr> MemoryStorage::getTransactions(
```

Comment:
Please don't move MemoryStorage::getTransactions location to make review more easy

Reply:
fixed it